.github/workflows/ci.yaml

```diff
@@ -88,6 +88,52 @@ jobs:
             ## Check that t_init is within [0, 20], and that there is only one value
             runalyzer -m summary.sqlite -c 'assert all([0 < r[0] < 20 for r in q("select $t_init.max")])'
             runalyzer -m summary.sqlite -c 'assert len([r[0] for r in q("select $t_init.max")]) == 1'
+        - name: Run and test example without gathering
+          run: |
+            set -x
+            python examples/log.py
```

Comment:
Can you test this without rerunning the example, and comparing the results of the autogathered values with the gathered file?

Reply:
Also, maybe you can gather and autogather multiple runs together and compare the output.